### PR TITLE
add rest api changes

### PR DESCRIPTION
**Pull request by gomathyk**
Files changed - 

supporting-email-account-verification-for-an-updated-email-address.md
application-rest-api.md
claim-management-rest-api.md
email-templates-rest-api.md
identity-governance-rest-api.md
idp-rest-api.md

applications.yaml
claim-management.yaml
email-template.yaml
identity-governance.yaml
idp.yaml

### CODE REVIEW

File: en/docs/develop/restapis/email-template.yaml [hunk 1]
```diff
@@ -103,9 +103,9 @@ paths:
     get:
       tags:
         - Email Template Types
-      summary: Retrieves the email template type corresponds to the template type id.
+      summary: Retrieves the email template type corresponding to the template type id.
```
Comment (sherenewso2) [inline en/docs/develop/restapis/email-template.yaml:106]: id --> ID?

File: en/docs/develop/restapis/email-template.yaml [hunk 2]
```diff
@@ -286,7 +286,7 @@ paths:
       summary: Retrieves a single email template.
       operationId: getEmailTemplate
       description: |
-        Retrieves the email template that matches to the template-type-id and the template-id.<br>
+        Retrieves the email template that matches the template-type-id and the template-id.<br>
```
Comment (sherenewso2) [inline en/docs/develop/restapis/email-template.yaml:289]: id --> ID?

File: en/docs/develop/supporting-email-account-verification-for-an-updated-email-address.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 #Support Email Account Verification for an Updated Email Address 
 
+This feature enables email account verification when the user updates the user profile with a new email address, so that the new email address can be taken into consideration for all further acrtivities performed by the user.
```
Comment (sherenewso2) [inline en/docs/develop/supporting-email-account-verification-for-an-updated-email-address.md:3]: acrtivities --> activities

File: en/docs/develop/restapis/email-template.yaml (hunk 2 above)
Comment (gomathyk) [reply en/docs/develop/restapis/email-template.yaml:289]: that's a property right?

**Pull request merged**